### ISSUE TITLE
XHK1-TC exit-delay - entry-delay not working, with XHK1-UE does work

### ISSUE THREAD

**Issue by thecobra666** [Koenkk/zigbee2mqtt#21474]
### What happened?

I have 2 keypads. One is from Universal Electronics Inc model XHK1-UE and the other is from Technicolor model XHK1-TC

The UE from universal starts beeping when using the entry or exit-delay. The TC does not. The TC does respond to all other modes, but not these. I can't find out why. I presume something is off in the configuration and not my automations since all the rest is working fine. 

No error is also trown in the zigbee2mqtt logs when asking it for the entry/delay mode. 

### What did you expect to happen?

Make it beep :) 

### How to reproduce it (minimal and precise)

Topic publish to zigbee2mqtt/keypadboven/set with package: {	"arm_mode": {    "mode": "exit_delay (or entry_delay)"	}}

Works for the UE, not for the TC model.

I've made a copy of the current convertor and changed the following line:

                globalStore.hasValue(device.getEndpoint(1), 'panelStatus')) {
                const payload = {
                    panelstatus: globalStore.getValue(device.getEndpoint(1), 'panelStatus'),
                    secondsremain: **0x1E**, audiblenotif: **0x01**, alarmstatus: 0x00,
                };

More specifically the secondsremain and audiblenotif value to 30 and 1. Those correspond with the values in ZHA cluster command. Then the keypad does starts to beep (for 30 seconds).

The problem with the code below however is that when setting it to entry_delay, it's also changing the arm mode to arm_day_zones?

Concluding: the TC needs the secondsremain and audiblenotif to enable the beep. The first one for the duration of the beep and the second to make it "audible". Hoping someone could figure out the definitive solution. This would solve it for a lot of people.

The same command for with ZHA, then it starts beeping. 

Entry_delay command:
service: zha.issue_zigbee_cluster_command
data:
ieee: xxxxxxxxxxxxxxxxxxxxxxx
cluster_id: 1281
command: 4
command_type: client
cluster_type: out
args: [5, 10, 1, 0]
endpoint_id: 1

The only issue is that the armed mode is switched one item to the left, why that is I have no clue. 

### Zigbee2MQTT version

1.35.3

### Adapter firmware version

0x26720700

### Adapter

Conbee

### Setup

Docker container

### Debug log

Debug 2024-02-19 16:36:11Received MQTT message on 'zigbee2mqtt/keypadboven/set' with data '{ "arm_mode": { "mode": "exit_delay" }}'
Debug 2024-02-19 16:36:11Publishing 'set' 'arm_mode' to 'keypadboven'
Debug 2024-02-19 16:36:14Received Zigbee message from 'keypadboven', type 'commandGetPanelStatus', cluster 'ssIasAce', data '{}' from endpoint 1 with groupID null
Info 2024-02-19 16:36:14MQTT publish: topic 'zigbee2mqtt/keypadboven', payload '{"action":null,"action_code":null,"action_transaction":null,"action_zone":null,"battery":100,"battery_low":false,"contact":true,"last_seen":"2024-02-19T15:36:18.580Z","linkquality":255,"occupancy":true,"presence":null,"tamper":true,"temperature":18,"voltage":6700}'

**Comment by thecobra666** [comment]
I want to add a little here there is indeed a difference with how the keypad is responding.

The UE model does this when arming the exit_delay / entry_delay etc..

`Debug ug 2024-02-24 09:18:56Received MQTT message on 'zigbee2mqtt/0x000d6f0014a6fe0a/set' with data '{ "arm_mode": { "mode": "entry_delay" } }'
Debug 2024-02-24 09:18:56Publishing 'set' 'arm_mode' to '0x000d6f0014a6fe0a'
Debug 2024-02-24 09:19:07Received MQTT message on 'zigbee2mqtt/0x000d6f0014a6fe0a/set' with data '{ "arm_mode": { "mode": "disarm" } }'
Debug 2024-02-24 09:19:07Publishing 'set' 'arm_mode' to '0x000d6f0014a6fe0a'

`

The TC does this:

Debug 2024-02-24 09:19:57Received MQTT message on 'zigbee2mqtt/keypadboven/set' with data '{ "arm_mode": { "mode": "exit_delay" } }'
Debug 2024-02-24 09:19:57Publishing 'set' 'arm_mode' to 'keypadboven'
**Debug 2024-02-24 09:19:58Received Zigbee message from 'keypadboven', type 'commandGetPanelStatus', cluster 'ssIasAce', data '{}' from endpoint 1 with groupID null**
Info 2024-02-24 09:19:58MQTT publish: topic 'zigbee2mqtt/keypadboven', payload '{"action":null,"action_code":null,"action_transaction":null,"action_zone":null,"battery":100,"battery_low":false,"contact":true,"last_seen":"2024-02-24T08:19:33.313Z","linkquality":255,"occupancy":true,"presence":null,"tamper":true,"temperature":17,"voltage":6700}'
Info 2024-02-24 09:21:28MQTT publish: topic 'zigbee2mqtt/keypadboven', payload '{"action":null,"action_code":null,"action_transaction":null,"action_zone":null,"battery":100,"battery_low":false,"contact":true,"last_seen":"2024-02-24T08:19:33.313Z","linkquality":255,"occupancy":false,"presence":null,"tamper":true,"temperature":17,"voltage":6700}'

They keypad responds back, so I'm guessing it's asking for a response and that's why it's not working.

**Comment by thecobra666** [comment]
Updating a little more. So I've added the keypad to ZHA and tried again. With ZHA I've found how to send a cluster command 

service: zha.issue_zigbee_cluster_command
data:
  ieee: xxxxxxxxxxxxxxxxxxxxxxx
  cluster_id: 1281
  command: 4
  command_type: client
  cluster_type: out
  args: [5, 10, 1, 0]
  endpoint_id: 1

And then the keypad does beep when it's armed for away mode. So the device is okay, the bug is in zigbee2mqtt. 

Probably the same command would work when it's arming for exit_delay

service: zha.issue_zigbee_cluster_command
data:
  ieee: xxxxxxxxxxxxxxxxxxxxxxx
  cluster_id: 1281
  command: 4
  command_type: client
  cluster_type: out
  args: [4, 10, 1, 0]
  endpoint_id: 1

@Koenkk who would be able to modify the settings for this device in z2m?
